atom/browser/api/lib/app.coffee

```diff
@@ -25,6 +25,12 @@ if process.platform is 'darwin'
     show: bindings.dockShow
     setMenu: bindings.dockSetMenu
 
+app._setAppPath = (path) ->
```

Comment:
I think we should use just `setAppPath` instead of `_setAppPath`, the underline is usually added when it is a native method and can not be called directly without a JavaScript wrap, and it is fine to have some undocumented APIs.